[TH]_Psycho_Pass_THE_MOVIE.ass

```diff
@@ -33,25 +35,32 @@ Dialogue: 0,0:00:45.58,0:00:47.92,Default,,0,0,0,,สิ่งที่รอค
 Dialogue: 0,0:00:49.38,0:00:50.59,Default,,0,0,0,,พวกมันกำลังมา
 Dialogue: 0,0:00:56.43,0:00:57.68,Default,,0,0,0,,ซ้าย 3 คลิก
 Dialogue: 0,0:00:58.76,0:01:00.27,Default,,0,0,0,,ขวา 4 คลิก
-Dialogue: 0,0:01:01.39,0:01:04.60,Default,,0,0,0,,ซ้าย 1 คลิก\Nขวา 2 คลิก
+Dialogue: 0,0:01:01.39,0:01:02.76,Default,,0,0,0,,ซ้าย 1 คลิก
+Dialogue: 0,0:01:03.12,0:01:04.66,Default,,0,0,0,,ขวา 2 คลิก
 Dialogue: 0,0:01:07.23,0:01:07.94,Default,,0,0,0,,ตอนนี้แหละ
 Dialogue: 0,0:01:09.94,0:01:11.61,Default,,0,0,0,,เยี่ยมมาก โควกามิ
-Dialogue: 0,0:01:21.74,0:01:25.58,Default,,0,0,0,,เราจะบอกว่ามันมีกฏของศิลปะที่โหดร้ายอยู่\N(ผู้แต่งบทกวีนี้กำลังพูดถึงเรื่องช่วงเวลาชีวิตของคน กับสิ่งที่มีค่า)\N (ตอนทำซับตรงนี้น่าจะใส่คำอธิบายหน่อยว่าเป็นบทกวี)
-Dialogue: 0,0:01:25.79,0:01:27.92,Default,,0,0,0,,นั่นคือมนุษย์ทุกคนล้วนต้องตายจากโลกนี้ไป\N(ก่อนที่จะสร้างผลงานที่มีคุณค่า)
-Dialogue: 0,0:01:28.21,0:01:31.25,Default,,0,0,0,,นั่นทำให้พวกเราต้องตายตกตามกันไป\N(ที่มีค่าพอที่โลกต้องจารึกไว้ก่อนที่จะล่วงลับไป)
-Dialogue: 0,0:01:31.42,0:01:34.76,Default,,0,0,0,,หลังจากทนทุกข์ทรมานทุกรูปแบบ\N(จะต้องผ่านความยากลำบากมานับไม่ถ้วน)
-Dialogue: 0,0:01:35.13,0:01:39.60,Default,,0,0,0,,ไม่ใช่หญ้าแห่งการลืมเลือน แต่เป็นหญ้าอันเป็นนิรันด์\N(เพื่อสร้างผลงานที่เลื่องชื่อ ไม่ใช่ผลงานที่ถูกปล่อยให้ลืม)
-Dialogue: 0,0:01:39.97,0:01:43.02,Default,,0,0,0,,ที่จะเติบโตและออกดอกออกผลต่อไป \N(เพื่อเป็นอนุสรณ์ให้คนรุ่นหลังต่อไป)
+Dialogue: 0,0:01:21.74,0:01:25.58,Default - Top,,0,0,0,,ผู้แต่งบทกวีนี้กำลังพูดถึงเรื่องช่วงเวลาชีวิตของคน กับสิ่งที่มีค่า
+Dialogue: 0,0:01:21.74,0:01:25.58,Default,,0,0,0,,เราจะบอกว่ามันมีกฏของศิลปะที่โหดร้ายอยู่
+Dialogue: 0,0:01:25.79,0:01:27.92,Default - Top,,0,0,0,,ก่อนที่จะสร้างผลงานที่มีคุณค่า
+Dialogue: 0,0:01:25.79,0:01:27.92,Default,,0,0,0,,นั่นคือมนุษย์ทุกคนล้วนต้องตายจากโลกนี้ไป
+Dialogue: 0,0:01:28.21,0:01:31.25,Default - Top,,0,0,0,,ที่มีค่าพอที่โลกต้องจารึกไว้ก่อนที่จะล่วงลับไป
+Dialogue: 0,0:01:28.21,0:01:31.25,Default,,0,0,0,,นั่นทำให้พวกเราต้องตายตกตามกันไป
+Dialogue: 0,0:01:31.42,0:01:34.76,Default - Top,,0,0,0,,จะต้องผ่านความยากลำบากมานับไม่ถ้วน
+Dialogue: 0,0:01:31.42,0:01:34.76,Default,,0,0,0,,หลังจากทนทุกข์ทรมานทุกรูปแบบ
+Dialogue: 0,0:01:35.13,0:01:39.60,Default - Top,,0,0,0,,เพื่อสร้างผลงานที่เลื่องชื่อ ไม่ใช่ผลงานที่ถูกปล่อยให้ลืม
+Dialogue: 0,0:01:35.13,0:01:39.60,Default,,0,0,0,,ไม่ใช่หญ้าแห่งการลืมเลือน แต่เป็นหญ้าอันเป็นนิรันด์
+Dialogue: 0,0:01:39.97,0:01:43.02,Default - Top,,0,0,0,,เพื่อเป็นอนุสรณ์ให้คนรุ่นหลังต่อไป
+Dialogue: 0,0:01:39.97,0:01:43.02,Default,,0,0,0,,ที่จะเติบโตและออกดอกออกผลต่อไป
 Dialogue: 0,0:02:04.00,0:02:05.62,Default,,0,0,0,,สำรวจความพร้อมของอุปกรณ์
 Dialogue: 0,0:02:12.25,0:02:13.63,Default,,0,0,0,,เปิดใช้งานแว่นตานำทาง
```

Comment:
กูฟังผิด มันพูดว่า "How was the glasses"

รบกวนมึงแก้เป็น "แว่นใช้งานได้มั้ย ?"